script/hassfest/manifest.py

```diff
@@ -6,13 +6,29 @@
 
 from .model import Integration
 
+DOCUMENTATION_URL_PREFIX = "https://www.home-assistant.io/integrations/"
+DOCUMENTATION_URL_EXCEPTIONS = ["https://www.home-assistant.io/hassio"]
+
 SUPPORTED_QUALITY_SCALES = [
     "gold",
     "internal",
     "platinum",
     "silver",
 ]
 
+
+def documentation_url(value: str) -> str:
```

Comment:
I intentionally didn't locate this in the config_validation package because I didn't want to clutter it with the two constants and I don't think this validator will need to be reused anywhere else.